src/main/java/codeu/model/PusherProvider.java

```diff
@@ -0,0 +1,39 @@
+package codeu.model;
+
+import com.pusher.rest.Pusher;
+
+public class PusherProvider {
+
+  /** The string identifier on web sockets for chat channel */
+  public static final String CHAT_CHANNEL= "chat";
+
+  /** The string identifier on web sockets for activity channel */
+  public static final String ACTIVITY_CHANNEL = "activity";
+
+  /** The string identifier on web sockets for new activity event */
+  public static final String NEW_ACTIVITY = "new-activity";
+
+  /** The string identifier on web sockets for message sent event*/
+  public static final String MESSAGE_SENT = "message-sent";
+
+  /** The key for out pusher app */
+  public static final String PUSHER_KEY = "1845b37e6fec77ec1a14";
+
+  /** The singleton Pusher instance*/
+  private static final Pusher pusher;
+
+  /** Initialization of the Pusher instance */
+  static {
+    pusher = new Pusher("547303", PUSHER_KEY, "922690b9fa9b5664364c");
```

Comment:
The app secret probably shouldn't be stored in the repo for security reasons. You can use something like git-crypt to encrypt the key in a special file and read it or load it as an environment variable and read it from there.